__tests__/schema/content-generation.ts

```diff
@@ -0,0 +1,125 @@
+import gql from 'graphql-tag'
+import { rest } from 'msw'
+
+import { user as baseUser } from '../../__fixtures__'
+import { Client, given, nextUuid } from '../__utils__'
+
+const mockPythonServiceResponse = JSON.stringify({
```

Comment:
```suggestion
const mockContentGenerationServiceResponse = JSON.stringify({
```
More descriptive name